modules/ServerAPI/src/client/Thing.coffee

```diff
@@ -387,13 +387,21 @@ class window.Thing extends Backbone.Model
 		copiedThing.get('scientist')?.set value: "unassigned"
 		copiedThing.get('completion date')?.set value: null
 
-		delete copiedThing.attributes.firstLsThings
-
 		secondItxs = copiedThing.get('secondLsThings')
 		secondItxs.each (itx) =>
 			@resetClonedAttrs(itx)
 			itxStates = itx.get('lsStates')
 			@resetStatesAndVals itxStates
+
```

Comment:
Change for another feature that got accidently added to this pull request.  This is an optional argument (still defaults to removing first lsThingInteractions) but allows you to choose to keep them if you want.